spec/CloudCode.spec.js

```diff
@@ -3531,6 +3531,124 @@ describe('afterLogin hook', () => {
 });
 
 describe('saveFile hooks', () => {
```

Comment:
Are these test correctly under the `saveFile hooks` or should it be a new `describe` section?